cloudwatch/cloudwatch.go

```diff
@@ -179,6 +192,13 @@ func NewOutputPlugin(config OutputPluginConfig) (*OutputPlugin, error) {
 		return nil, err
 	}
 
+	runningInECS := true
+	ecsTaskMetadataEndpointV3 := os.Getenv("ECS_CONTAINER_METADATA_URI")
+	// check if it is running in ECS
+	if ecsTaskMetadataEndpointV3 == "" {
+		runningInECS = false
```

Comment:
You don't need the variable `ecsTaskMetadataEndpointV3` here. You can just do:

```
if os.Getenv("ECS_CONTAINER_METADATA_URI") == "" {
```

Reply:
Thanks for the suggestion